ts/Core/Tooltip.ts

```diff
@@ -503,6 +503,7 @@ class Tooltip {
         if (this.renderer) {
             this.renderer = this.renderer.destroy() as any;
             discardElement(this.container as any);
+            this.container = void 0 as any;
```

Comment:
Looks like `container` is wrongly typed when we have to cast `void 0` to any. The type should be `HTMLElement|undefined`. But we may not need this change if we get the above to work.